charts/redpanda/testdata/ci/32-statefulset-podspec-novalues.yaml.golden

```diff
@@ -654,23 +672,14 @@ spec:
         - name: kafka-default-cert
           secret:
             defaultMode: 272
-            items:
```

Comment:
Still not sure where this change is coming from and it doesn't seem like it's correct/desirable.

Reply:
It is desirable as per https://github.com/redpanda-data/helm-charts/pull/1402#discussion_r1661911665

I didn't want to list all items explicitly, but rather mount secret and relay on the (implicit) fact that TLS secrets generated by cert-manager and our test suite will have always ca.crt, tls.crt and tls.key.

Reply:
SGTM! I missed and misunderstood your first comment.